frontend/public/locales/en/apply.json

```
@@ -48,7 +48,8 @@
       "personal-information": {
         "heading": "Personal Information",
         "service-canada-application-administration": "The personal information collected in this application is required for the purposes of applying for the Canada Dental Care Plan (CDCP). Service Canada, which is part of Employment and Social Development Canada (ESDC), administers applications for the CDCP on behalf of Health Canada. Service Canada will collect the personal information you include in your CDCP application to register and enroll you into the Plan.",
-        "collection-use": "The collection, use, and disclosure of your personal information is authorized by, among other laws, the Department of Health Act, the Dental Care Measures Act, the Department of Employment and Social Development Act, and is protected by the provisions of the Privacy Act. ESDC carries out this application process using services from a third-party provider, Microsoft Corporation, which may be subject to foreign laws. For more information, see <microsoftDataPrivacyPolicy>Microsoft Data Privacy Policy & Privacy Principles</microsoftDataPrivacyPolicy>."
+        "collection-use": "The collection, use, and disclosure of your personal information is authorized by, among other laws, the <i>Department of Health Act</i>, the <i>Dental Care Measures Act</i>, the <i>Department of Employment and Social Development Act</i>, and is protected by the provisions of the <i>Privacy Act</i>.",
```

Referencing the title of a legal document should be done using the [citation element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite)

```suggestion
        "collection-use": "The collection, use, and disclosure of your personal information is authorized by, among other laws, the <cite>Department of Health Act</cite>, the <cite>Dental Care Measures Act</cite>, the <cite>Department of Employment and Social Development Act</cite>, and is protected by the provisions of the <cite>Privacy Act</cite>.",
```

(Note that this same comment can be applied wherever `<i>` is used).

---

I don't disagree but [Canada.ca - Terms and conditions](https://www.canada.ca/en/transparency/terms.html) doesn't use `cite` html tag.

---

Interesting. Seems like that Canada.ca page doesn't follow the  [Canada.ca Content Style Guide](https://design.canada.ca/style-guide/), which uses `cite` throughout.

---

Updated to use `<cite>` tags